beancount_import/source/__init__.py

```diff
@@ -255,7 +255,6 @@ class Source:
     """
 
     def __init__(self, log_status: Callable[[str], None], **kwargs) -> None:
-        super().__init__(**kwargs)  # type: ignore
```

Comment:
Why was this removed?

Reply:
When we pass in an additional parameter (for example `currency` in this case) to the data source initialization dictionary, thie line reports the following error:

```
  File "/Users/jason/.pyenv/versions/3.8.11/lib/python3.8/site-packages/beancount_import/source/__init__.py", line 258, in __init__
    super().__init__(**kwargs)
TypeError: object.__init__() takes exactly one argument (the instance to initialize)
> /Users/jason/.pyenv/versions/3.8.11/lib/python3.8/site-packages/beancount_import/source/__init__.py(258)__init__()
-> super().__init__(**kwargs)
```

This is not specific to Mint importer. Adding any additional parameter to any the the importer will cause this error. Since the class `Source` doesn't have any parent class, it should be fine to remove this line.

Reply:
I don't like dropping the base class init call completely, but I can live with `super().__init__()`.

Reply:
Now keeping `super().__init__()`